src/geocoder.jl

```diff
@@ -8,6 +8,65 @@ Column names in the geonames dumpfile (from http://download.geonames.org/export/
 const COLUMNS = [:geonameid, :name, :asciiname, :alternatenames, :latitude, :longitude, 
                 :feature_class, :feature_code, :country_code, :cc2, :admin1_code, :admin2_code, 
                 :admin3_code, :admin4_code, :population, :elevation, :dem, :timezone, :modification_date]
+
+const COLUMN_TYPE = Dict(
+    :geonameid => Int, 
+    :name => String, 
+    :asciiname => String, 
+    :alternatenames => String, 
+    :latitude => Float64, 
+    :longitude => Float64, 
+    :feature_class => String, 
+    :feature_code => String, 
+    :country_code => String, 
+    :cc2 => String, 
+    :admin1_code => String, 
+    :admin2_code => String, 
+    :admin3_code => String, 
+    :admin4_code => String, 
+    :population => Int, 
+    :elevation => Int,
+    :dem => Int, 
+    :timezone => String, 
+    :modification_date => String
+)
+
+function _extract_column_values(row::DataFrameRow, headers)::Vector{Any}
+    values = Vector{Any}(undef, length(headers))
+    
+    for (i, header) ∈ enumerate(headers)
+        value = getproperty(row, header)
+        type  = COLUMN_TYPE[header]
+
+        if ismissing(value)
+            value = type == String ? "" : 0
+        elseif type == String
```

Comment:
I am not sure what is happening here; why is `type==String` checked twice?

Reply:
Ah this is really a `ismissing` vs. not, and in the case of . 

There's a way to avoid this by having some default behaviors on the `CSV.read` for missing entries. I don't mind turning this into an issue then fixed in another PR so that we don't PR scope increase too much

Reply:
Sure, sorry, I think I had a brain cramp and blacked out the `ismissing` when reading this. However, if you want to modify the CSV flags in another PR, by all means :-)